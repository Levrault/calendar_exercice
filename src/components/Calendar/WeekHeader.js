import React from 'react';
import styles from './WeekHeader.css';

const WeekHeader = () => {
  return (
    <ul className={styles.header}>
      <li className={styles.item}>S</li>
      <li className={styles.item}>M</li>
      <li className={styles.item}>T</li>
      <li className={styles.item}>W</li>
      <li className={styles.item}>T</li>
      <li className={styles.item}>F</li>
      <li className={styles.item}>S</li>
    </ul>
  );
};

export default WeekHeader;
