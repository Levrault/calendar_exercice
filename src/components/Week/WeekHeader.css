.header {
  @mixin horizontalList;
  justify-content: space-between;
}

.item {
  color: $palette-secondary-text;
  display: flex;
  flex: 1;
  justify-content: center;
  margin: $space-m;
}
