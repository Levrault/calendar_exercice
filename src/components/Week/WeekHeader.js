import React from 'react';
import styles from './WeekHeader.css';

/**
 * Mont header that represent
 * every day of a week
 * e.g. S M T W T F S
 */
const WeekHeader = () => (
  <ul className={styles.header}>
    <li className={styles.item}>S</li>
    <li className={styles.item}>M</li>
    <li className={styles.item}>T</li>
    <li className={styles.item}>W</li>
    <li className={styles.item}>T</li>
    <li className={styles.item}>F</li>
    <li className={styles.item}>S</li>
  </ul>
);

export default WeekHeader;
