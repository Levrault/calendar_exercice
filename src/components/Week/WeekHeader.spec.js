import React from 'react';
import renderer from 'react-test-renderer';
import WeekHeader from './WeekHeader';

describe('WeekHeader component', () => {
  it('should render without problem', () => {
    const component = renderer.create(
      <WeekHeader />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
