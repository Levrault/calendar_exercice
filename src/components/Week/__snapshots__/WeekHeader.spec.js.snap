// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeekHeader component should render without problem 1`] = `
<ul>
  <li>
    S
  </li>
  <li>
    M
  </li>
  <li>
    T
  </li>
  <li>
    W
  </li>
  <li>
    T
  </li>
  <li>
    F
  </li>
  <li>
    S
  </li>
</ul>
`;
